bin/bin.js

```diff
@@ -287,6 +289,22 @@ const main = async () => {
       }
     }
 
+    if (exec) {
+      const execCmd = exec
+        .replace("{}", _path.resolve(outputPodcastPath))
```

Comment:
What do you think about swapping these `replace` calls to a regex that replaces all in case someone needs to access the path twice?

```suggestion
        .replace("/{}/g", _path.resolve(outputPodcastPath))
```

Same idea for `{filenameBase}`. 👍

Reply:
done in f51afda